Add deregister user documentation (#129)
input-output-hk/partner-chains: docs/user-guides/registered.md

4. Obtain chain parameters from the chain builder
5. Register for the partner chain
6. Run the partner chain node

> 7. Optionally: deregister from the partner chain

Reviewer: ```suggestion
7. Optional: deregister from the partner chain
```

Registration is effective after 1-2 Cardano epochs. After the waiting period, the partner chain node is registered on the partner chain and is a selection option for the consensus committee.

> ### 7. Optionally: deregister from the partner chain

Reviewer: ```suggestion
### 7. Optional: deregister from the partner chain
```

> To deregister from the block producers candidates, you need to run the deregister wizard.

Reviewer: ```suggestion
To deregister from the list of block producer candidates, you need to run the deregister wizard.
```
